### ISSUE TITLE
Clean up "effective module" errors while building with mvn

### ISSUE THREAD

**Issue by kwsutter** [eclipse-ee4j/jakartaee-api#93]
The following messages are logged when attempting to build this API component.  This log is from our official jenkins builds, but you can easily reproduce it with a simple command line build as well.

```
+ mvn -B -Pstaging,oss-release -DskipTests -Ddoclint=none -DstagingRepositoryId=jakartaplatform-1013 -Dmaven.javadoc.failOnError=false deploy
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for jakarta.platform:jakarta.jakartaee-bom:pom:9.0.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-enforcer-plugin is missing. @ org.eclipse.ee4j:project:1.0.6, /home/jenkins/.m2/repository/org/eclipse/ee4j/project/1.0.6/project-1.0.6.pom, line 164, column 29
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for jakarta.platform:jakarta.jakartaee-web-api:jar:9.0.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-enforcer-plugin is missing. @ org.eclipse.ee4j:project:1.0.6, /home/jenkins/.m2/repository/org/eclipse/ee4j/project/1.0.6/project-1.0.6.pom, line 164, column 29
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for jakarta.platform:jakarta.jakartaee-api:jar:9.0.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-enforcer-plugin is missing. @ org.eclipse.ee4j:project:1.0.6, /home/jenkins/.m2/repository/org/eclipse/ee4j/project/1.0.6/project-1.0.6.pom, line 164, column 29
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for jakarta.platform:jakartaee-api-parent:pom:9.0.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-enforcer-plugin is missing. @ org.eclipse.ee4j:project:1.0.6, /home/jenkins/.m2/repository/org/eclipse/ee4j/project/1.0.6/project-1.0.6.pom, line 164, column 29
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```

**Comment by scottkurz** [comment]
I took a look and expect this is already fixed in:

**Comment by scottkurz** [comment]
Or I guess switch to use the 1.0.7-SNAPSHOT

**Comment by pzygielo** [comment]
> Or I guess switch to use the 1.0.7-SNAPSHOT

Where one could get 1.0.7-SNAPSHOT of that parent?

**Comment by kwsutter** [comment]
Should be resolved with PR #98 (with the creation of the 1.0.7 parent pom).
